Build angular-cli fork fails

> Please provide us with the following information:
> ---------------------------------------------------------------

### OS?
> Windows 10.

### Versions.
> angular-cli: 1.0.0-beta.26
node: 5.11.0
os: win32 ia32
@angular/compiler: 2.4.5
@angular/compiler-cli: 2.4.5
@angular/core: 2.4.5
@angular/tsc-wrapped: 0.5.1


### Repro steps.
1. I have downloaded beta26 angular-cli, from here: https://github.com/angular/angular-cli/releases/tag/v1.0.0-beta.26
2. npm install - success
3. npm run build - fails


### The log given by the failure.

```angular-cli@1.0.0-beta.26 build C:\Users\Fuit Attila\Desktop\angular-cli-1.0.0-beta.26
node ./scripts/publish/build.js

Deleting dist folder...
Creating schema.d.ts...
module.js:341
    throw err;
    ^

Error: Cannot find module 'C:\Users\Fuit'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Function.Module.runMain (module.js:447:10)
    at startup (node.js:148:18)
    at node.js:405:3
Command failed: node C:\Users\Fuit Attila\Desktop\angular-cli-1.0.0-beta.26\scripts\build-schema-dts.js C:\Users\Fuit At
tila\Desktop\angular-cli-1.0.0-beta.26\packages\angular-cli\lib\config\schema.json C:\Users\Fuit Attila\Desktop\angular-
cli-1.0.0-beta.26\packages\angular-cli\lib\config\schema.d.ts
module.js:341
    throw err;
    ^

Error: Cannot find module 'C:\Users\Fuit'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Function.Module.runMain (module.js:447:10)
    at startup (node.js:148:18)
    at node.js:405:3


npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-
cli.js" "run" "build"
npm ERR! node v5.11.0
npm ERR! npm  v3.8.6
npm ERR! code ELIFECYCLE
npm ERR! angular-cli@1.0.0-beta.26 build: node ./scripts/publish/build.js
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the angular-cli@1.0.0-beta.26 build script 'node ./scripts/publish/build.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the angular-cli package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./scripts/publish/build.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs angular-cli
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls angular-cli
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Fuit Attila\Desktop\angular-cli-1.0.0-beta.26\npm-debug.log
PS C:\Users\Fuit Attila\Desktop\angular-cli-1.0.0-beta.26
```


> ---------------------------------------------------------------
What am I doing wrong?

Thank you for your help!

@catull 
I have moved my project to here: C:\WORK\.... It worked, thank you!

But now I have an other failure:

```
> angular-cli@1.0.0-beta.26 build C:\WORK\angular-cli-1.0.0-beta.26
> node ./scripts/publish/build.js

Deleting dist folder...
Creating schema.d.ts...
Compiling packages...
  @angular-cli\ast-tools
Compilation error.
node_modules/@types/lodash/index.d.ts(245,25): error TS2304: Cannot find name 'Partial'.
```

Any idea what am I doing wrong? Maybe I shoud install some typings globally - unit it will be fixed?

--- a/scripts/publish/build.js
+++ b/scripts/publish/build.js
@@ -61,7 +61,7 @@ Promise.resolve()
     const script = path.join(root, 'scripts/build-schema-dts.js');
     const input = path.join(root, 'packages/angular-cli/lib/config/schema.json');
     const output = path.join(root, 'packages/angular-cli/lib/config/schema.d.ts');
-    return npmRun.execSync(`node ${script} ${input} ${output}`);
+    return npmRun.execSync(`node "${script}" "${input}" "${output}"`);
   })
   .then(() => console.log('Compiling packages...'))
   .then(() => {
@@ -87,7 +87,7 @@ Promise.resolve()
         return promise.then(() => {
           console.log(`  ${name}`);
           try {
-            return npmRun.execSync(`tsc -p ${path.relative(process.cwd(), pkg.root)}`);
+            return npmRun.execSync(`tsc -p "${path.relative(process.cwd(), pkg.root)}"`);
           } catch (err) {
             throw new Error(`Compilation error.\n${err.stdout}`);
           }

